### DISCUSSION TITLE
How to inference with multiple GPUs?

### DISCUSSION THREAD

**Discussion by jiamying** [open-mmlab/mmocr#1723] [Q&A] [answered]
I tried using `--device`, but it seems that it can only be deployed on a single GPU. Is there a way to run inference on multiple GPUs?

**Comment by gaotongxiao** [comment, marked as answer]
Currently not - a workaround is to use [multi-gpu test](https://mmocr.readthedocs.io/en/dev-1.x/user_guides/train_test.html#training-and-testing-with-multiple-gpus) and set the `--save-preds` flag, which allows you to access the prediction results later.
